src/ansys_sphinx_theme/search/fuse_search.py

```diff
@@ -204,7 +204,17 @@ def create_search_index(app, exception):
 
     search_index_list = []
 
-    for document in app.env.found_docs:
+    static_search_options = app.config.html_theme_options.get("static_search", {})
+    excluded_docs = static_search_options.get("files_to_exclude", [])
+
+    for docs in excluded_docs:
+        included_docs = [
+            doc
+            for doc in app.env.found_docs
+            if not (doc.startswith(docs) if docs.endswith("/") else doc == docs)
```

Comment:
Could we refactor this line? The logic is tricky here. If we could use boolean variables explaining the idea that would be great.

Reply:
sure, added